packages/ui/src/components/vuestic-components/va-slider/VaSlider.vue

```diff
@@ -245,14 +244,14 @@ export default class VaSlider extends Mixins(
   get dotClass () {
     if (this.range) {
       return [
-        { 'va-slider__handler--inactive': !this.hasMouseDown },
-        { 'va-slider__handler--inactive': !this.hasMouseDown },
+        { 'va-slider__handler--inactive': !this.isFocused },
+        { 'va-slider__handler--inactive': !this.isFocused },
       ]
     }
 
     return {
-      'va-slider__handler--on-focus': !this.range && (this.flag || this.isKeyboardFocused),
-      'va-slider__handler--inactive': !this.hasMouseDown,
+      'va-slider__handler--on-focus': !this.range && (this.flag || this.isFocused),
```

Comment:
Focus and keyboard focus are kinda different things.
Focus we don't really want to visualize in any way for slider. Keyboard focus - we do.